integration/examples/react-reload/skaffold.yaml

```diff
@@ -5,9 +5,8 @@ build:
   - image: gcr.io/k8s-skaffold/react-reload
     context: app
     sync:
-      'src/components/*': src/components/
```

Comment:
oops.. this should not have happened (@.myself). The reason why it could is that `react-reload` has no test-case in `run_test.go`.